lib/v2/screens/send_confirmation/interactor/mappers/send_transaction_state_mapper.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/screens/profile_screens/profile/interactor/usecases/get_profile_use_case.dart';
+import 'package:seeds/v2/screens/send_confirmation/interactor/viewmodels/send_confirmation_commands.dart';
+import 'package:seeds/v2/screens/send_confirmation/interactor/viewmodels/send_confirmation_state.dart';
+
+class SendTransactionStateMapper extends StateMapper {
+  SendConfirmationState mapResultToState(SendConfirmationState currentState, Result result) {
+    if (result.isError) {
+      return currentState.copyWith(pageState: PageState.failure, error: result.asError.error.toString());
+    } else {
+      return currentState.copyWith(
+          pageState: PageState.success, pageCommand: NavigateToTransactionSuccess(result.asValue.value.toString()));
```

Comment:
page command, @RaulUrtecho  
This is to let the page know it needs to do something thats not necessarily draw.
In this case its navigate, But it could be to show a bottom sheet, or open the camera, etc.

Reply:
Oh I see. I guess this page Command can also work with an enum.

Reply:
it could, but often times we need data as well in the commands